vmaas_sync/vmaas_sync.go

```diff
@@ -112,6 +112,11 @@ func websocketHandler(data []byte, _ *websocket.Conn) error {
 			utils.Log("err", err.Error()).Fatal("Failed to sync advisories")
 		}
 
+		err = syncRepos()
```

Comment:
It would be good to extend TestSync to ensure that repos marking works properly. You can set "repo1" third_party = true in the beginning of the test and then check it was set to false during the sync.